src/actions/menu_items_actions.ts

```diff
@@ -436,10 +436,27 @@ export const OPEN_CF_SIDEPANEL_ACTION = (env: SpreadsheetChildEnv) => {
 };
 
 export const INSERT_LINK = (env: SpreadsheetChildEnv) => {
-  let { col, row } = env.model.getters.getActivePosition();
+  const { col, row } = env.model.getters.getActivePosition();
   env.model.dispatch("OPEN_CELL_POPOVER", { col, row, popoverType: "LinkEditor" });
 };
 
+export const INSERT_CHECKBOX = (env: SpreadsheetChildEnv) => {
+  const { sheetId, col, row } = env.model.getters.getActivePosition();
+  const ranges = [env.model.getters.getRangeDataFromXc(sheetId, toXC(col, row))];
+  env.model.dispatch("ADD_DATA_VALIDATION_RULE", {
+    ranges,
+    sheetId,
+    rule: {
+      id: env.model.uuidGenerator.uuidv4(),
+      criterion: {
+        type: "isBoolean",
+        values: [],
+      },
+    },
+  });
+  env.model.dispatch("UPDATE_CELL", { sheetId, col, row, content: "FALSE" });
```

Comment:
two dispatches means two history steps (you would need to hit Ctrl+Z twice to undo the change). We might need to add a new "helper" command to do that. e.g. INSERT_CHECKBOX

Reply:
After a discussion with Adrien, we decided to change the way Checkbox is added in the datavalidation rule and always add "FALSE" as the default content